docs/updates/2020-08-07-announce-book.md

```diff
@@ -0,0 +1,108 @@
+# 2020-08-07 - Announcing the new Jupyter Book
+
+*Note: this announcement is cross-posted between the [Jupyter Blog](https://blog.jupyter.org) and the [Executable Book Project updates blog](https://executablebooks.org/en/latest/updates/index.html)*
+
+Jupyter Book is an open source project for building beautiful, publication-quality books and documents from computational material. With this post, we're happy to announce that Jupyter Book has been re-written from the ground up, making it easier to install, faster to use, and able to create more complex publishing content in your books. It is now supported by [the Executable Book Project](https://executablebooks.org/en/latest/), an open community that builds open source tools interactive and executable documents in the Jupyter ecosystem and beyond.
```

Comment:
"from computational material" sounds a bit odd to me, what does it mean?
(note: in the "What next?" section you use "computational content")

Maybe something like:

Jupyter Book is an open source project for combining text and data into beautiful, publication-quality books and documents.

Reply:
"builds open source tools interactive" is there some words missing after tools? "builds open source tools for interactive"?

Reply:
I updated this so it includes "source material with computational content" per @jstac's suggestion above